spec/unit/shoulda/matchers/active_record/association_matchers/model_reflection_spec.rb

```diff
@@ -92,13 +92,15 @@
         context 'when the scope is a block' do
           it 'executes the block in the context of an empty scope' do
             define_model(:country, mood: :string)
-            person_model = define_model(:person, country_id: :integer) do
-              belongs_to :country, -> { where(mood: 'nice') }
+            columns = { country_id: :integer, spirit: :string }
+            person_model = define_model(:person, columns) do
+              belongs_to :country, ->(person) { where(mood: person.spirit) }
```

Comment:
I also thought the same, previously I committed as such, but it looked so much as a duplication that I removed. I will put it back.